### ISSUE TITLE
"Unhandled promise rejection" on Safari 11

### ISSUE THREAD

**Issue by JohnBain** [mozmorris/react-webcam#60]
On Safari 11 (Mac OS X High Sierra) webcam does not show up and this error displays in console:

```
[Error] Unhandled Promise Rejection: TypeError: Type error
	createObjectURL (bundle.js:104017)
	handleUserMedia (bundle.js:104017)
	(anonymous function) (bundle.js:103957)
	promiseReactionJob

```

Debugger shows this file as the source of **Unhandled** Promise Rejection: **TypeError**:
```
var src = window.URL.createObjectURL(stream);
```

**Comment by cliener** [comment]
I implemented the new API as part of #42. Scoped npm install: https://www.npmjs.com/package/@cliener/react-webcam

**Comment by mozmorris** [comment]
I believe this error has now be fixed. I'll reopen again if there's still a problem.
